saleor/product/models/base.py

```diff
@@ -235,6 +241,7 @@ def get_first_category(self):
 
     def is_available(self):
         # type: () -> bool
+        """Checks if product is available today for customers"""
```

Comment:
But this depends on current date. Item can be unavailable because of missing stock, but this filter does not check that.